src/components/MMSearch/MMSearch.component.tsx

```diff
@@ -186,10 +200,15 @@ export const MMSearch = (props: MMSearchProps) => {
             className={`mm-autocomplete ${className}`}
         >
             <Input
-                ref={inputRef}
+                ref={handleSetRef}
                 fullWidth={fullWidth}
                 searchQuery={searchQuery}
-                onKeyDown={onKeyDown}
+                onKeyDown={(e) => {
+                    onKeyDown(e);
+                    if (onKeyPress) {
+                        onKeyPress(e);
```

Comment:
any reason we need to do both?

Reply:
yes @avas27JTG , suppose the user wants to add any functionality on `onKeyPress` then he can pass it here. In our case (open ai) we need to override default behaviour of hitting enter for a text area (which goes to the next row) to on hitting enter should send the API request

Reply:
cool